### PR TITLE
handle unhandled error on follow_users method

### PR DESCRIPTION
**Pull request by AfshinZlfgh**
delay_in_seconds added to delay.py for making delay in wanted time
unhandled error(404 and other errors except 400 and 429) handled in follow_users, 400 and 429 handled in API before.

### CODE REVIEW

**Comment by ohld** [comment]
Thanks for the commit! Can you please fix Travis CI errors and ill merge that

**Comment by AfshinZlfgh** [comment]
i read Travis CI log and fixed its error, but i cant fix the last one and i dont know its meaning! if you help me and tell how i can fix it, i will fix it. thank you

File: instabot/bot/bot_follow.py [hunk 1]
```diff
@@ -45,9 +45,31 @@ def follow_users(self, user_ids):
         user_ids))  # Log to console
     for user_id in tqdm(user_ids, desc='Processed users'):
         if not self.follow(user_id):
+            if self.LastResponse.status_code == 404:
+                print("due 404 error user " + str(user_id) + " doesn't exist. \nskip follow user id:" + str(user_id))
+                broken_items.append(user_id)
+
+            # other unhandled error except 400(block to follow) and 429(many request error) like 500 error
+        elif self.LastResponse.status_code != 400 and self.LastResponse.status_code != 429:
```
Comment (sudoguy) [inline instabot/bot/bot_follow.py:53]: Wrong indentation level. Fix to the same level as line 48

File: instabot/bot/bot_follow.py [hunk 2]
```diff
@@ -45,9 +45,31 @@ def follow_users(self, user_ids):
...
+                try_number = 3
+                i = 0
+                error_pass = False
+                for i in range(0, try_number):
```
Comment (sudoguy) [inline instabot/bot/bot_follow.py:57]: in python you can use just ```range(try_number)```
start use 0 as default value

**Review by sudoguy — CHANGES_REQUESTED** [review]

**Comment by sudoguy** [comment]
Closed, because @morozru was resolved all conflicts in https://github.com/instagrambot/instabot/pull/435 based on your decision.
Thanks, @AfshinZlfgh!

**Pull request closed without merging**